### ISSUE TITLE
Idee: Erweitere Einstellungen (Nether erlauben, Seed, Rcon usw.) vereinfacht erklären

### ISSUE THREAD

**Issue by pavl21** [gnmyt/VoxelDash#70]
![image](https://github.com/gnmyt/MCDash/assets/36771750/9c5ab476-e959-4415-9937-17a51cd07bb9)

Derzeit wird in dieser Übersicht alles so gezeigt wie in der properties Datei, glaube ich. Um Neueinsteiger noch ein bisschen mehr mit MCDash zu helfen, fände ich es vorteilhaft und hilfreich, wenn da genauer steht was es macht und daunter in die Beschreibung was es "ganz genau" macht.

----------------------

Bsp.:
Nether erlauben
Lässt Spieler in das Nether teleportieren, wenn sie über ein Netherportal dorthin gelangen wollen.

Fliegen erlauben
Erlaubt Spielern ohne Operator-Rechte mehr oder länger zu Fliegen als üblich.

----------------------

**Comment by gnmyt** [comment]
Yes, versteh ich. Problem ist hierbei, dass die Daten, wie du schon gesagt hast, einfach aus der Properties Datei übernommen werden. Ist dementsprechend bisschen schwieriger, trotzdem noch machbar. Ich halte dich hier auf dem laufenden :)

**Comment by gnmyt** [comment]
<img width="1909" height="949" alt="Image" src="https://github.com/user-attachments/assets/660b240d-63b8-4b38-938d-ebfda54e5aea" />

Die neuste Version beinhaltet Erklärungen für unterschiedliche Properties. Nicht direkt alle, da immer neue dazu kommen, jedoch einige.
